My-Portfolio.html

```diff
@@ -0,0 +1,41 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div>
+      <header id="header">
+        <div class="flex-container">
+          <nav id="nav-bar">
+            <div id="logo-placeholder">
+              <h2>John Doe</h2>
+            </div>
+            <div id="normal-button">
+              <img id="menu-button" src="icons/Icon - Menu.svg" alt="menu-button"/>
+            </div>
+          </nav>
+        </div>
+      </header>  
+      <div>
+        <section id="headline">
+          <p id="primary-text">Hey There. I'm Eduardo</p>
+          <p id="supporting-text">I'am a Software Developer</p>
+          <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+          <div class="flex-container" id="social-media">
+            <img id="logo1" src="icons/logo1.svg" alt="logo1" />
+            <img id="logo2" src="icons/logo2.svg" alt="logo2" />
+            <img id="logo3" src="icons/logo3.svg" alt="logo3" />
+            <img id="logo4" src="icons/logo4.svg" alt="logo4" />
+            <img id="logo5" src="icons/logo5.svg" alt="logo5" />
+          </div>
+          <img id="background-decoration" src="icons/Illustration.svg" alt="decoration" />
```

Comment:
- [ ] [**REQUIRED**] This image is a background image for the headline section and should not be added to the screen with an `img` tag, but with CSS `background-image` property. There is a rule of thumb, if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.